### ISSUE TITLE
Communication Suggestion: Update the Pull Request Template

### ISSUE THREAD

**Issue by HarshCasper** [moja-global/moja_global_docs#96]
### Describe how we are communicating and why it is not optimal.

The current Pull Request template is highly skewed towards development projects, not for documentation related work. You can view the [existing PR template](https://github.com/moja-global/moja_global_docs/blob/master/.github/PULL_REQUEST_TEMPLATE.md).

We would want it to be updated for the purpose of reporting documentation changes more appropriately. 

### Describe the solution you'd like.

We can add the following sections to the Pull Request template: 

- Issue reference (NA if not available)
- PR Description
- SME Reviewer (Tag the GitHub user)
- Docs reviewer (Tag the GitHub user)
- Preview Links (Open the RTD build and find where the docs is rendered and put the link)

**Comment by aornugent** [comment]
IMO - our current PR template is too much. Maybe we can go for something simpler and more general that covers the majority of potential use cases?

PRs need to:
1. describe the changes
2. not break anything
3. close an issue
4. ?

**Comment by shraddhavp** [comment]
@aornugent  and @HarshCasper  would like to contribute to the issue

**Comment by HarshCasper** [comment]
> IMO - our current PR template is too much. Maybe we can go for something simpler and more general that covers the majority of potential use cases?
> 
> PRs need to:
> 
> 1. describe the changes
> 2. not break anything
> 3. close an issue
> 4. ?

I agree. The CI parts take care if something breaks or not. For the Docs part, it would be necessary to have a look at the rendered docs though, as it makes things easy for us to review. I will make an issue on `About_moja_global` for the PR template update.
